src/helpers/formatScreenSizes.js

```diff
@@ -0,0 +1,9 @@
+//Opdracht 2c: helpferfunctie die een string genereert voor alle schermgroottes van 1 tv
+
+export function formatScreenSizes(sizes) {
+    //Functie om inches naar centimeters om te rekenen
+    const inchesToCm = (inches) => (inches * 2.54).toFixed(1);
```

Comment:
inches heeft hier geen waarde waardoor deze formule niet werkt. Hierdoor krijg je ook alleen de string te zien op de pagina en niet de uitkomst van de berekening. In de return hieronder gebruik je keurig de map methode waardoor je de berekening in die uitkomst had kunnen plaatsen: